features/other_a/preferences_edit_more.feature

```diff
@@ -48,3 +48,42 @@ Feature: Preferences
   Given I am logged out
     And I am logged in as "tasteful"
     And I go to the "Big and Loud" work page
+
+  Scenario: Hidden users' user pages are disallowed for search engine indexing
+    Given the user "hidden" exists and is activated
+      And the user "hidden" is hidden from search engines
+    When I go to hidden's user page
+    Then the page should be hidden from search engines
+
+  Scenario: Unhidden users' user pages are allowed for search engine indexing
+    Given the user "unhidden" exists and is activated
+    When I go to unhidden's user page
+    Then the page should not be hidden from search engines
+
+  Scenario: Hidden users' works pages are disallowed for search engine indexing
+    Given the user "hidden" exists and is activated
+      And the user "hidden" is hidden from search engines
+      And the work "Hidden Work" by "hidden"
+    When I view the work "Hidden Work"
+    Then the page should be hidden from search engines
+
+  Scenario: Unhidden users' works pages are allowed for search engine indexing
+    Given the user "unhidden" exists and is activated
+      And the work "Unhidden Work" by "unhidden"
+    When I view the work "Unhidden Work"
+    Then the page should not be hidden from search engines
+
+  Scenario: Hidden users' series pages are disallowed for search engine indexing
```

Comment:
I think you can combine these 6 tests into just 2, one hidden one default, since their setup is similar.

After `I post the work "Hidden Work" as part of a series "Hidden Series"` you can check the user dashboard, the work, and the series in one go.